### PR TITLE
More detailed instructions on releases

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: ci/release_setup.md [hunk 1]
```diff
@@ -76,3 +76,88 @@ Versioned releases can be one of two kinds:
...
+    git push origin
+    ```
+
+1.  Create a PR for your branch based on the `release-X.Y` branch.
+
+1.  Once the PR is merged, it will be included in the next minor release, which is
+    usually built Tuesday nights, between 2AM and 3AM.
+
+**Note**: If a minor release is required for a release branch that's not the latest,
+the job must be [started manually](https://github.com/knative/test-infra/blob/master/ci/manual_release.md#creating-a-dot-release-on-demand).
```
Comment (chaodaiG) [inline ci/release_setup.md]: why not use relative link?

Comment (adrcunha) [reply ci/release_setup.md]: Because of copy&paste :) Fixed

**Pull request merged**